artificial-scrum-master/apps/artificial.scrum.master/Artificial.Scrum.Master.ScrumIntegration/ScrumIntegrationEndpoints.cs

```diff
@@ -29,7 +31,10 @@ public static void RegisterScrumIntegrationEndpoints(this IEndpointRouteBuilder
                 var userId = userAccessor.UserId;
                 var result = await service.Handle(userId);
 
-                await context.Response.WriteAsJsonAsync(result);
+                await context.Response.WriteAsJsonAsync(result, new JsonSerializerOptions
```

Comment:
Te Opcje możesz wynieść jako statyczne pole w tej klasie endpoints według mnie. Będzie tylko raz zainicjowany i jedno dla wszystkich połączeń :)

Reply:
A druga sprawa po co ci te opcje w ogóle?

Reply:
Myślałem że mi łatwiej będzie wyciągać dane na froncie jeśli w ValuesDiff nie będzie propert z nullami (tak jak w response z Taigi) ale w sumie to bez znaczenia chyba jednak. To wywalę te opcje.

Reply:
EJ chwila a te modele valueddiff i milestone to skąd są? Nie przecieka nam infrastruktura zewnętrznego serwisu do api?